packages/lesswrong/lib/editor/mentionsConfig.tsx

```diff
@@ -34,8 +34,7 @@ const linkPrefix = getSiteUrl()
 const initSearchForIndex = (indexName: AlgoliaIndexCollectionName) => {
   const searchClient = getSearchClient()
   const index = searchClient.initIndex(getAlgoliaIndexName(indexName))
-  const search = (...args: AnyBecauseTodo[]) => index.search(...args)
-  return promisify(search)
+  return promisify(index.search.bind(index));
```

Comment:
Hm... I think this should be `return promisify(index.search).bind(index)`?  `index.search.bind(index)` seems like a no-op to me - which, admittedly, preserves the previous logic; it's just not clear to me that the previous logic was correct.  (Of course, if losing `this` context had mattered, I'd expect to have already been seeing errors caused by that, so maybe not?  But if the previous logic was fine, is this change required?)

Reply:
I must've had a brain fart, idk why I thought `index.search.bind(index)` was a no-op.